edx_exams/apps/router/interop.py

```diff
@@ -23,10 +29,58 @@ def register_exams(course_id, exam_list):
         response = make_request('PATCH', url, client, json=exam_list)
     except HTTPError as e:
         response = e.response
-    return response
 
+    response_data = _get_json_data(response)
```

Comment:
the LMS doesn't always return JSON when things go wrong this makes a mess of things on our end. Wrap all these responses so we aren't logging or returning junk html.

Reply:
Nice, really like this enhancement